### PR TITLE
fix: InfoSQLDatabaseTool on openai functions mode sometimes calling tables instead of table_names as function arg

### PR DESCRIPTION
**Pull request by puigde**
**Description:** 
Improve the description for the InfoSQLDatabaseTool in order to ensure that the llm guesses the input properly on openai functions mode.

### CODE REVIEW

**Comment by baskaryan** [comment]
thanks for pr @puigde! for context, what's openai function output you're seeing?

**Comment by puigde** [comment]
> thanks for pr @puigde! for context, what's openai function output you're seeing?

Hi @baskaryan it depended but most times the agent tried to pass the argument as 'tables' instead of 'table_names' raising a invalid args for the function call.

File: libs/langchain/langchain/tools/sql_database/tool.py [hunk 1]
```diff
@@ -52,6 +52,7 @@ class InfoSQLDatabaseTool(BaseSQLDatabaseTool, BaseTool):
     Input to this tool is a comma-separated list of tables, output is the schema and sample rows for those tables.    
 
     Example Input: "table1, table2, table3"
+    The arg to pass the string of tables to the function is table_names.
```
[the remark this answers is not in the document]
Comment (puigde) [reply libs/langchain/langchain/tools/sql_database/tool.py]: Updated, 

If you'd wish another format please let me know :):)

**Comment by AlexCapchase** [comment]
Any updates on this?

**Comment by puigde** [comment]
> Any updates on this?
Looks like sql tools stuff had a path change bringing merge conflicts. Now should be ok for merge afaik. cc. @baskaryan, @hwchase17 ??

**Comment by baskaryan** [comment]
resolved in #16756

**Pull request closed without merging**